Apps/Sandcastle/gallery/Callback Position Property.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
+    />
+    <meta
+      name="description"
+      content="Use Viewer to start building new applications or easily embed Cesium into existing applications."
```

Comment:
Let's make this more relevant to the example. Something like:

```suggestion
      content="Use a CallbackPositionProperty when your data can't be pre-computed or needs to be derived from other properties at runtime."
```

Reply:
Yes !